### PR TITLE
Feature/menu and misc

### PR DESCRIPTION
**Pull request by fruniia**
Merge Menu from MenuAndMisc-branch into main.

### CODE REVIEW

**Comment by JonnaJK** [comment]
We should write tests before merging :)

File: PathsOfPower.Cli/Program.cs
Comment (friborg) [inline PathsOfPower.Cli/Program.cs:1]: This should be removed in the future!

Comment (JonnaJK) [reply PathsOfPower.Cli/Program.cs:1]: Yes boss!

**Review by friborg — APPROVED** [review]
Good job :)

**Pull request merged**